staging/src/k8s.io/kms/scripts/encryption-config.yaml

```diff
@@ -0,0 +1,10 @@
+kind: EncryptionConfiguration
+apiVersion: apiserver.config.k8s.io/v1
+resources:
+  - resources:
+      - secrets
+    providers:
+      - kms:
+          apiVersion: v2
+          name: kmsprovider
+          endpoint: unix:///tmp/kms.socket
```

Comment:
We already have encryption configuration in https://github.com/kubernetes/kubernetes/tree/master/test/e2e/testing-manifests/auth/encrypt. Shouldn't that one be modified instead of adding a new one?